MeshLib/MeshEditing/AddLayerToMesh.cpp

```diff
@@ -31,6 +32,7 @@ namespace MeshLib
 
 MeshLib::Element* extrudeElement(std::vector<MeshLib::Node*> const& subsfc_nodes,
 	MeshLib::Element const& sfc_elem,
+	MeshLib::PropertyVector<int> const& sfc_to_subsfc_id_map,
```

Comment:
See commit 9712bd762ceb81778303c6585484586ffe15c37e for type change (`int` -> `size_t`) and 9712bd7 for documentation.